cluster-install/README.adoc

```diff
@@ -74,23 +76,23 @@ The Kops CLI can be used to create a highly available cluster, with multiple mas
 - Setting up AWS resources such as networks, Auto Scaling groups, IAM users, and security groups
 - Installing Kubernetes.
 
-Start the Kubernetes cluster using the following command:
+Create a simple Kubernetes cluster using the following command. The cluster name suffix of `.k8s.local` will result in the gossip protocol being used for masters/nodes to discover each other. Note that `create cluster` only creates and stores the cluster config in S3; you must use `update cluster` to build the cluster:
 
     kops create cluster \
-      --name cluster01.kubernetes-aws.io \
+      --name cluster01.k8s.local \
```

Comment:
Is gossip-based cluster a GA feature now? We should keep the cluster to use Route 53, by default.